docs/tutorials.rst

```diff
@@ -24,6 +24,7 @@ The following tutorials explore various optimization problems using Trieste.
    :maxdepth: 1
 
    notebooks/expected_improvement
+   notebook/batch_optimization
```

Comment:
(the docs build step will have caught this once you'd resolved conflicts and merged up)

```suggestion
   notebooks/batch_optimization
```